examples/account_management/link_manager_to_client.py

```diff
@@ -16,13 +16,9 @@
 
 
 import argparse
-import sys
-import uuid
-from datetime import datetime, timedelta
-from google.api_core import protobuf_helpers
 
 from google.ads.google_ads.client import GoogleAdsClient
-from google.ads.google_ads.errors import GoogleAdsException
+from google.api_core import protobuf_helpers
```

Comment:
This import statement should be above `GoogleAdsClient` and separated by a new line because it's from an external module. See the section [here](https://www.python.org/dev/peps/pep-0008/#imports):

```
Imports should be grouped in the following order:

* Standard library imports.
* Related third party imports.
* Local application/library specific imports.

You should put a blank line between each group of imports.
```

So:

```
import argparse

from google.api_core import protobuf_helpers

from google.ads.google_ads.client import GoogleAdsClient
```

Reply:
Fixed.